.github/workflows/kernel-demos.yml

```diff
@@ -154,8 +159,8 @@ jobs:
       - name: Checkout the FreeRTOS/FreeRTOS Repository
         uses: actions/checkout@v3
         with:
-          ref: main
-          repository: FreeRTOS/FreeRTOS
+          ref: ARM_CRx_MPU
+          repository: skptak/FreeRTOS
```

Comment:
Remember to switch this

Reply:
I'll remove the build check from this PR and we can add it in after the [FreeRTOS/FreeRTOS PR](https://github.com/FreeRTOS/FreeRTOS/pull/1149) gets merged